src/main/java/wolox/training/constants/PreconditionsMessages.java

```diff
@@ -0,0 +1,9 @@
+package wolox.training.constants;
+
+public class PreconditionsMessages {
```

Comment:
👏👏👏 I would only change this class name to `PreconditionsConstans`  or `PreconditionsMessagesConstants`, because in all the projects i've been in wolox, they used that naming convention for constants' classes.

Reply:
now the class is renamed, thanks!